src/stan-users-guide/latent-discrete.Rmd

```diff
@@ -1019,3 +1019,182 @@ the probability mass function of $z_i$ in the posterior.   These may
+evaluating $\textsf{E}(g(X, Y) \mid X)$ becomes tractable.
+
+If $g(X, Y)$ is a function of only the continuous parameters $X$, the 
+conditional expectation simplifies:
+
+$$
+\textsf{E}(g(X, Y) \mid X) 
+= \textsf{E}(g(X) \mid X) = g(X).
+$$
+
+Our conditional estimator therefore becomes:
+
+$$
+\frac{1}{M} \sum_{i = 1}^M g(x_i).
+$$
+
+This makes performing inference straightforward. First, we write a Stan program 
+that implements the marginalized posterior $p_X(x)$. Running this model will 
+produce samples from the marginalized posterior. We then evaluate $g(\cdot)$ at
+each sample, and use the average over the samples to estimate the expectation 
+of interest. That is, we use Stan `as usual', except that we implement a
+marginalized posterior.
+
+### Recovering the discrete parameters {-}
+
+For the discrete parameters, we are often only interested in the marginal 
+posterior mass function.  For example, for a univariate $Y$ we want to 
+calculate $\Pr(Y = k)$ for all possible values of $k$; note that this allows 
+us to calculate the posterior of any quantity of interest involving $Y$. To 
+estimate $\Pr(Y = k)$ under the marginalization scheme we can note that
+
+$$
+\Pr(Y = k) = \textsf{E}(\mathbf{1}(Y = k)),
+$$
+
+in other words that $g(X, Y) = \mathbf{1}(Y = k)$. Then, from the properties of conditional expectation we have
+
+$$
+h(X) 
+= \textsf{E}(g(X, Y) \mid X) 
+= \textsf{E}(\mathbf{1}(Y = k) \mid X) 
+= \Pr(Y = k \mid X).
+$$
+
+We can therefore estimate $\Pr(Y = k)$ with
+
+$$
+\frac{1}{M} \sum_{i = 1}^M \Pr(Y = k \mid X = x_i).
+$$
+
+When calculating this conditional probability it is important to remember that
+we are also conditioning on the observed data, $D$. That is, we are really
+estimating $\Pr(Y = k \mid D)$ with
+
+$$
+\frac{1}{M} \sum_{i = 1}^M \Pr(Y = k \mid X = x_i, D).
+$$
+
+This point is important as it suggests one of the main ways of calculating the required conditional probability. Using Bayes' theorem gives us:
+
+$$
+\Pr(Y = k \mid X = x_i, D) 
+= \frac{               \Pr(D \mid Y = k, X = x_i) \Pr(Y = k \mid X = x_i)}
+       {\sum_{k = 1}^K \Pr(D \mid Y = k, X = x_i) \Pr(Y = k \mid X = x_i)}.
+$$
+
+Here, $\Pr(D \mid X = x_i, Y = k)$ is the likelihood conditional on a 
+particular value of the latent variables. Crucially, all elements of the 
+expression can be calculated using the samples from the posterior of the 
+continuous parameters and knowledge of the model structure. Note that for 
+certain models, such as [zero-inflated count models](#zero-inflated.section), 
+some values of the latent parameters and data may be incompatible, giving probabilities of exactly zero.
+
+Other than the use of Bayes' theorem, $\Pr(Y = k \mid X = x_i, D)$ can also be extracted by coding up the Stan model to include the conditional probability explicitly, as is done for the [Dawid and Skene model](#data-coding-models.section).
+
+### Summary {-}
```

Comment:
We don't need summaries in the manual.  These sections are super short.  If there are points you need to make, please put them in the text.  Here, I think this can all just be dropped if you move the citation and mention of Rao-Blackwell up (perhaps even its own section).